app/models/user.js

```diff
@@ -10,7 +10,8 @@ module.exports = (sequelize, DataTypes) => {
       firstName: { type: DataTypes.STRING, allowNull: false },
       lastName: { type: DataTypes.STRING, allowNull: false },
       email: { type: DataTypes.STRING, allowNull: false, unique: true },
-      password: { type: DataTypes.STRING, allowNull: false }
+      password: { type: DataTypes.STRING, allowNull: false },
+      admin: { type: DataTypes.BOOLEAN, allowNull: false }
```

Comment:
Maybe we should set `false` as defaultValue for this column, just in case. What do you think?